subxt/src/tx/tx_payload.rs

```diff
@@ -138,7 +138,7 @@ impl<'a> TxPayload for DynamicTxPayload<'a> {
         let pallet = metadata.pallet(&self.pallet_name)?;
         let call_id = pallet.call_ty_id().ok_or(MetadataError::CallNotFound)?;
         let call_value =
-            Value::unnamed_variant(self.call_name.to_owned(), self.fields.clone());
+            Value::unnamed_variant(self.call_name.clone(), self.fields.clone());
```

Comment:
clippy didn't like that because `to_owned` is called on `Cow`, thus clippy suggests either `into_owned` or `clone (which should be cheaper)`